web/src/lib/components/photos-page/actions/remove-from-shared-link.svelte

```diff
@@ -4,17 +4,26 @@
   import { handleError } from '$lib/utils/handle-error';
   import { removeSharedLinkAssets, type SharedLinkResponseDto } from '@immich/sdk';
   import { mdiDeleteOutline } from '@mdi/js';
-  import ConfirmDialogue from '../../shared-components/confirm-dialogue.svelte';
   import { NotificationType, notificationController } from '../../shared-components/notification/notification';
   import { getAssetControlContext } from '../asset-select-control-bar.svelte';
+  import { DialogType, dialogController } from '$lib/components/shared-components/dialog/dialog';
 
   export let sharedLink: SharedLinkResponseDto;
 
-  let removing = false;
-
   const { getAssets, clearSelect } = getAssetControlContext();
 
   const handleRemove = async () => {
+    const isConfirmed = await dialogController.show({
+      type: DialogType.Confirm,
+      title: 'Remove assets?',
+      prompt: `Are you sure you want to remove {getAssets().size} asset(s) from this shared link?`,
```

Comment:
You could use the new s function here if you wanted to pluralise asset(s) in the correct cases